go.mod

```diff
@@ -1,3 +1,13 @@
 module github.com/bytecodealliance/wasmtime-go
 
-go 1.14
+go 1.18
+
+require github.com/stretchr/testify v1.8.0
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
```

Comment:
Is it necessary to list out indirect dependencies here?

Reply:
These were added by `go mod tidy`.  

It looks like some can be removed but others are necessary and need to stay.

Reply:
Ah ok sounds good, I'll leave these as-is then.